src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File file;
+        file = new File(fileName);
+        for (Byte oneByte : data) {
+            try (FileOutputStream fileOutputStream = new FileOutputStream(file, true)) {
```

Comment:
let's not open the new outputstream for each byte
we have a method to write the whole byte array

Reply:
Removed...